code/game/turfs/snow.dm

```diff
@@ -78,9 +78,15 @@
 	if(slayer > 0)
 		if(iscarbon(AM))
 			var/mob/living/carbon/C = AM
-			C.next_move_slowdown += (isxeno(C) ? 0.25 : 0.5) * slayer
+			C.next_move_slowdown += 0.5 * slayer
 			if(prob(1))
 				to_chat(C, "<span class='warning'>Moving through [src] slows you down.</span>")
+			if(isxeno(C))
+				C.next_move_slowdown -= 0.25 * slayer
+				var/mob/living/carbon/xenomorph/X = C
```

Comment:
this applies the slowdown twice

Reply:
![nonsense](https://user-images.githubusercontent.com/64715958/118712301-7bed5300-b7d5-11eb-998a-607133dea604.gif)

Reply:
![cat gif](https://user-images.githubusercontent.com/57223640/118713264-03f94a00-b822-11eb-9bc4-ab2380d94dbe.gif)